testsuite/documentation/cucumber-steps.md

```diff
@@ -289,7 +289,7 @@ The radio button can be identified by name, id or label text.
 ```gherkin
   When I check "manageWithSSH"
   When I uncheck "role_org_admin"
-  When I check "Container Build Host" if not checked
```

Comment:
just as a side note: this line is about check boxes, not radio buttons - but the correction is still correct.